examples/Example/babel.config.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
         extensions: ['.js', '.ts', '.tsx'],
         alias: {
           'react-native-gesture-handler': '../../',
```

Comment:
I'd suggest to specify the source file explicitely for this one too

Reply:
Fixed in 1d0b0b8.

Also, moved index to src/